hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/InvocationBuilder.java

```diff
@@ -1,268 +1,277 @@
-     */
-    public InvocationBuilder setDoneCallback(Runnable doneCallback) {
-        this.doneCallback = doneCallback;
-        return this;
-    }
-
-    public abstract <E> InvocationFuture<E> invoke();
-}
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.spi.impl.operationservice;
+
+import com.hazelcast.core.IndeterminateOperationStateException;
+import com.hazelcast.internal.server.ServerConnectionManager;
+import com.hazelcast.internal.partition.InternalPartition;
+import com.hazelcast.cluster.Address;
+import com.hazelcast.internal.serialization.Data;
+import com.hazelcast.spi.impl.operationservice.impl.InvocationFuture;
+
+import static com.hazelcast.internal.util.Preconditions.checkFalse;
+import static com.hazelcast.spi.impl.operationservice.Operation.GENERIC_PARTITION_ID;
+
+/**
+ * The InvocationBuilder is responsible for building an invocation of an operation and invoking it.
+ * <p>
+ * The original design exposed the actual Invocation class, but this will limit flexibility since
+ * the whole invocation can't be changed or fully removed easily.
+ */
+public abstract class InvocationBuilder {
+
+    /**
+     * Default call timeout.
+     */
+    public static final long DEFAULT_CALL_TIMEOUT = -1L;
+
+    /**
+     * Default replica index.
+     */
+    public static final int DEFAULT_REPLICA_INDEX = 0;
+
+    /**
+     * Default try count.
+     */
+    public static final int DEFAULT_TRY_COUNT = 250;
+
+    /**
+     * Default try pause in milliseconds. If a call is retried, then perhaps a delay is needed.
+     */
+    public static final long DEFAULT_TRY_PAUSE_MILLIS = 500;
+
+    /**
+     * True that the result of an operation automatically should be deserialized to an object.
+     */
+    public static final boolean DEFAULT_DESERIALIZE_RESULT = true;
+
+    protected final String serviceName;
+    protected final Operation op;
+    protected final int partitionId;
+    protected final Address target;
+    protected Runnable doneCallback;
+
+    protected long callTimeout = DEFAULT_CALL_TIMEOUT;
+    protected int replicaIndex;
+    protected int tryCount = DEFAULT_TRY_COUNT;
+    protected long tryPauseMillis = DEFAULT_TRY_PAUSE_MILLIS;
+    protected boolean resultDeserialized = DEFAULT_DESERIALIZE_RESULT;
+    protected boolean failOnIndeterminateOperationState;
+    protected boolean async = false;

```

Comment:
This flag + setter is the only change in this class. I haven't launched auto-formatting etc., so don't really know why there are so many changes in this file. Git diff shows only correct changes.

Reply:
Looks like its the line endings, the changed file has `CRLF` instead of `LF`. The PR build should actually fail because of this - there's a checkstyle rule for verifying that all line endings are only `LF`.

Reply:
@ivanthescientist Thanks, changing to LF back worked :)